source/_integrations/roomba.markdown

```diff
@@ -60,3 +60,9 @@ docker exec -it CONTAINER_NAME_OR_ID python -c 'import roombapy.entry_points; ro
 The command to retrieve the credentials does not need any additional software to be installed because it uses the built-in [roombapy](https://github.com/pschmitt/roombapy) package and [password](https://github.com/pschmitt/roombapy/blob/1.6.1/roomba/entry_points.py#L20) function deployed with Home Assistant.
 
 </div>
+
+<div class='note'>
+  
+If the password cannot be retrieved, another smart home system may be using continuous mode. Ensure no other systems use your Roomba and try again. 
```

Comment:
This is a duplicate of the first note. This page already consists of big boxes, this seems a bit not needed.